gBrogue Source/gBrogueCode/Rogue.h

```diff
@@ -2857,6 +2869,8 @@ extern "C" {
 	boolean proposeOrConfirmLocation(short x, short y, char *failureMessage);
 	boolean useStairs(short stairDirection);
 	short passableArcCount(short x, short y);
+        boolean againstAWall(short x, short y);
```

Comment:
New gBrogue functions need to be declared in order for the game to compile on some compilers on normal settings.